specification/resources/resource-manager/Microsoft.Authorization/2017-06-01-preview/policy.json

```diff
@@ -0,0 +1,1102 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PolicyClient",
+    "version": "2017-06-01-preview",
```

Comment:
I don't understand. policyDefinitions APIs won't work with 2017-06-01-preview version?

Reply:
Correct, api-versions are resourceType specific and policyDefinitions did not have a change that the people who made the changes thought warranted a new api-version.  Calls to its APIs with 2017-06-01-preview will not work

Reply:
Ah, I see. Before I go ahead and do the split, shouldn't we bump the api-version for policyDefinitions as well, now that we have mode?

Reply:
Mode was already released to prod on all existing api-versions so it could be even more breaking to restrict it now (even though its probably not used by anyone yet).  There is the option of introducing a new api-version on policyDefinitions just to make this work easier but I don't know if that's a great idea.  It will be good to have them split in the future so they can be incremented individually when needed.  If everything gets a new api-version every time anything changes you end up with a ton of api-versions that user has to wade through.